tests/test_readFitsCatalog.py

```diff
@@ -98,6 +119,9 @@ def testHDU1DefaultNames(self):
         self.assertTrue(np.array_equal(self.arr1, table))
         self.assertEqual(len(table), 2)
 
+        schema = self._makeSchema(table.dtype)
+        self.assertIsNotNone(schema)
```

Comment:
I'm confused about what this `IsNotNone` test is supposed to do. `_makeSchema` can't return None, can it?